src/components/file/file.tsx

```diff
@@ -93,6 +106,22 @@ export class File {
     private chipSet;
     private mdcTextField;
 
+    private get chipArray() {
```

Comment:
I don't know if there's any benefit to using a getter as opposed to just creating the chip in the render function, apart from making the code look a little nicer?

It does expose the chip outside the component (since `private` is a typescript lie 😄), which I'm inclined to see as a negative.

I reverted this change in my first fixup, but that was mainly because I wanted to keep the changes from `master` to be minimal. I can always bring back the refactor to using a getter in a separate commit.

Thoughts @jgroth?

Reply:
There's no way for an outsider to get hold of the instance where all the methods are declared since the component instance (`this`) is not the same as the DOM element, so `private`/`public` does not really matter in most cases. To expose a method on the DOM element you will have to decorate it with `@Method`

Reply:
Hmm… yeah, I can't seem to get that node… but is that really not possible? If so, that's great!

I guess it's not really a big problem in this case anyway, so let's go with that getter 👍